cmd/cloud.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"context"
+	"os"
+
+	"github.com/snyk/policy-engine/pkg/input"
+	"github.com/snyk/policy-engine/pkg/input/cloudapi"
+	"github.com/snyk/policy-engine/pkg/models"
+	"github.com/spf13/cobra"
+)
+
+type cloudOptions struct {
+	OrgID         string
+	EnvIDs        []string
+	ResourceTypes []string
+	ResourceIDs   []string
+	NativeIDs     []string
+	IDs           []string
+	Platforms     []string
+	Names         []string
+	Locations     []string
+}
+
+func (c *cloudOptions) addFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&c.OrgID, "cloud.org", "", "Cloud organization ID (required to fetch cloud resources)")
```

Comment:
```suggestion
	cmd.PersistentFlags().StringVar(&c.OrgID, "cloud-org", "", "Cloud organization ID (required to fetch cloud resources)")
```

Minor superficial thing but I'd prefer to stick with `-` for flags... unless there's a good reason I'm not aware of.

Reply:
There's not a good reason 😄 in my mind it made it clearer that these arguments were grouped together, but I think you're right that it looks inconsistent. Fixed in c945332.